packages/calcite-components/src/components/stepper-item/stepper-item.scss

```diff
@@ -256,13 +297,12 @@
   }
 
   .stepper-item-header {
-    @apply border-color-3
-    border-0
+    @apply border-0
     border-solid
-    border-t-2
     focus-base;
     grid-row: items;
-
+    border-block-start-width: var(--calcite-spacing-base);
+    border-color: var(--calcite-color-border-3);
```

Comment:
I can't find a storybook example where this border is applied? Can you double check if this even does anything? if not let's remove it. If so, let's give it a component token.

Reply:
This redundant with https://github.com/Esri/calcite-design-system/pull/8906/files#diff-c77e15f7b2a230aaeb9146248b360a9ac7d9e7b867732486b0982ad26d59f43dR33. removed.